Finished lab

DeathofaSellout opened the pull request.

spragala commented on username.rb:
```diff
@@ -2,13 +2,107 @@
+      final_name = "admin-" + user_name + add_year
+    else
+      final_name = "" + user_name
+    end
```
This looks good but I'm getting some code smell. You've written the same piece of code 4 times. Think about ways to refactor this code where maybe you're calling methods you have already written.

DeathofaSellout commented:
Can you call other functions within a function?

On Wed, Sep 13, 2017 at 9:31 AM Matt Sprague <notifications@github.com>
wrote:

> *@spragala* commented on this pull request.
> ------------------------------
>
> In username.rb
> <https://github.com/SF-WDI-LABS/username-generator/pull/10#discussion_r138672134>
> :
>
> > +  else
> +    first_name = first.split
> +    last_name = last.split
> +    change_last_name = last_name.join()
> +    user_name = first_name[0][0].downcase + change_last_name.downcase
> +    if level.floor == 0
> +      final_name =  user_name + add_year
> +    elsif level.floor == 1
> +      final_name = "seller-" + user_name + add_year
> +    elsif level.floor == 2
> +      final_name = "manager-" + user_name + add_year
> +    elsif level >= 3
> +      final_name = "admin-" + user_name + add_year
> +    else
> +      final_name = "" + user_name
> +    end
>
> This looks good but I'm getting some code smell. You've written the same
> piece of code 4 times. Think about ways to refactor this code where maybe
> you're calling methods you have already written.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/SF-WDI-LABS/username-generator/pull/10#pullrequestreview-62499047>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AI2GqgKAWZzuKZXsHKCo48jxdXsRvboTks5siAN2gaJpZM4PWWmF>
> .
>
